types/hapi/index.d.ts

```diff
@@ -18,54 +18,3673 @@
 
 /// <reference types="node" />
 
-/** PLUGIN */
-export * from './definitions/plugin/plugin';
-export * from './definitions/plugin/plugin-registered';
-
-/** REQUEST */
-export * from './definitions/request/request';
-export * from './definitions/request/request-auth';
-export * from './definitions/request/request-events';
-export * from './definitions/request/request-info';
-export * from './definitions/request/request-route';
-
-/** RESPONSE */
-export * from './definitions/response/response-events';
-export * from './definitions/response/response-object';
-export * from './definitions/response/response-settings';
-export * from './definitions/response/response-toolkit';
-
-/** ROUTE */
-export * from './definitions/route/route-options';
-export * from './definitions/route/route-options-access';
-export * from './definitions/route/route-options-cache';
-export * from './definitions/route/route-options-cors';
-export * from './definitions/route/route-options-payload';
-export * from './definitions/route/route-options-pre';
-export * from './definitions/route/route-options-response';
-export * from './definitions/route/route-options-secure';
-export * from './definitions/route/route-options-validate';
-
-/** SERVER */
-export * from './definitions/server/server';
-export * from './definitions/server/server-auth';
-export * from './definitions/server/server-auth-scheme';
-export * from './definitions/server/server-cache';
-export * from './definitions/server/server-events';
-export * from './definitions/server/server-ext';
-export * from './definitions/server/server-info';
-export * from './definitions/server/server-inject';
-export * from './definitions/server/server-method';
-export * from './definitions/server/server-options';
-export * from './definitions/server/server-options-cache';
-export * from './definitions/server/server-realm';
-export * from './definitions/server/server-register';
-export * from './definitions/server/server-route';
-export * from './definitions/server/server-state';
-export * from './definitions/server/server-state-options';
-
-/** UTIL */
-export * from './definitions/util/common';
-export * from './definitions/util/json';
-export * from './definitions/util/lifecycle';
-export * from './definitions/util/util';
+import * as Boom from "boom";
+import * as catbox from "catbox";
+import * as http from "http";
+import * as https from "https";
+import * as Shot from "shot";
+import * as stream from "stream";
+import * as url from "url";
+import * as zlib from "zlib";
+
+import Podium = require("podium");
```

Comment:
Can also be typed 

```ts
import Podium from "podium";
```

as far as I know

Reply:
No it can't this is the change I asked jhsimms to make: https://github.com/Microsoft/TypeScript/issues/21872

Reply:
Sorry your correct that's my bad I think I have been up to long my eyes are playing tricks on me. Either is fine.

Reply:
The linter and the compiler are both angry if I try to change it from the above.  I left it as it, as there are a lot of other issues (linter) that arose from collapsing into a single file that I needed to address.